### PR TITLE
Add `content-cell-disabled-island` style

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

Add a utility class for sub/alternative islands.

# Changes

- add `content-cell-disabled-island` style

# Screenshots

image

### CODE REVIEW

File: src/lib/styles/global/utility-classes.scss [hunk 1]
```diff
@@ -46,6 +46,13 @@
   border-radius: var(--border-radius);
 }
 
+.content-cell-disabled-island {
```
Comment (lmuntaner) [inline src/lib/styles/global/utility-classes.scss:49]: Why "disabled"? It looks more highlighted to me.

Do we really need to do all this with the "content-cell..." utility classes?

Are we using them only for proposals?

Comment (mstrasinskis) [reply src/lib/styles/global/utility-classes.scss:49]: Yes, currently on for proposal summary. Why not, let me do it directly in nns-dapp.

Comment (lmuntaner) [reply src/lib/styles/global/utility-classes.scss:49]: I'm ok to move it. Maybe we just need components shared between NNS and SNS proposals with the CSS inside them.

Comment (mstrasinskis) [reply src/lib/styles/global/utility-classes.scss:49]: Currently, the ProposalSummary component is used for both. I suggest thinking about it when we need to reuse these styles.

**Comment by mstrasinskis** [comment]
Not necessary anymore

**Pull request closed without merging**